ui/apps/platform/src/Containers/ComplianceEnhanced/Schedules/ScanConfigsTablePage.tsx

```diff
@@ -240,7 +243,12 @@ function ScanConfigsTablePage({
                     </Td>
                     {isReportJobsEnabled && (
                         <Td dataLabel="My last job status">
-                            {/* TODO: Create the <MyLastComplianceReportJobStatus /> component and add it here */}
+                            <MyLastJobStatus
+                                snapshot={complianceReportSnapshot}
```

Comment:
Asking about naming convention, not necessarily requesting changes: comparing simplification of `reportSnapshot={reportSnapshot}` to `snapshot={snapshot}` in test file above, what is your thought for components:
* Here is compliance: `snapshot={complianceReportSnapshot}`
* Following in vulnerabilities: `snapshot={reportSnapshot}`

Find in Files is my friend and I am not sure whether even the more advanced features in Visual Studio Code will be able to find all occurrences in Compliance and Vulnerabilities. Although copy-paste is not the ideal method for code reuse, similar naming makes it smoother when it is needed.